### DISCUSSION TITLE
better flow entering split transactions

### DISCUSSION THREAD

**Discussion by bpaulien** [actualbudget/releases#123] [Feature ideas]
When I enter a split, and put in the price of the first item, and hit enter, it creates a new line (this part I like) however, the cursor remains in the Payment field, but at least the way I enter transactions, I like to put in the note and category first, and then the price, so it would be better if the active field on the new line was Notes.  That way I wouldn't have to shift+tab a couple times to get over there, and start entering the data for the new split category line.

**Comment by MikeBishop** [comment]
In the same vein, it would be nice to have the ability to auto-distribute splits.  This is useful for handling sale tax, shipping, discounts, etc.  You enter the item price for the different things, then let Actual scale amounts up or down proportionally to match the total.

**Reply by bpaulien** [reply]
So, I'm wondering, would it be helpful, or more annoying for it to auto populate the price of the new split with the remaining balance?  If it was the last split, you'd just hit enter to confirm.  If it wasn't you'd type in the amount for this split, and hit enter, creating another split, (also pre-populated with the remaining balance)...  Eventually it'll be right.  It seems to me like that would be helpful.  I wonder if others agree, or you think it would be a nuisance.
